docs/configuration-reference/platforms/aws.md

```diff
@@ -219,10 +221,11 @@ worker_pool "my-worker-pool" {
 | `region`                      | AWS region to use for deploying the cluster.                                                                                                                                               | "eu-central-1"  |    string    |  false   |
 | `enable_aggregation`          | Enable the Kubernetes Aggregation Layer.                                                                                                                                                   |      true       |     bool     |  false   |
 | `enable_tls_bootstrap`        | Enable TLS bootstraping for Kubelet.                                                                                                                                                       |      true       |     bool     |  false   |
+| `encrypt_pod_traffic`         | Enable in-cluster pod traffic. If true you need to reduce the default `network_mtu`, see its documentation.                                                                                |      false      |     bool     |  false   |
```

Comment:
This doesn't apply anymore. Also, the first sentence doesn't make sense. How about this?

```suggestion
| `encrypt_pod_traffic`         | Enable in-cluster pod traffic encryption. If true `network_mtu` is reduced by 40 to make room for the encryption header.                                                                                |      false      |     bool     |  false   |
```

Please change all the other platforms as well.